lib/fluent/plugin/filter_viaq_data_model.rb

```diff
@@ -90,6 +90,9 @@ class ViaqDataModelFilter < Filter
     desc 'Name of destination timestamp field'
     config_param :dest_time_name, :string, default: '@timestamp'
 
+    desc 'Use event transformations, available options ["none", "eventrouter"]'
+    config_param :event_trans, :string, default: 'none'
```

Comment:
If these are an enumerated list of options, use the `enum` type like this: https://github.com/ViaQ/fluent-plugin-viaq_data_model/blob/master/lib/fluent/plugin/filter_viaq_data_model.rb#L105